core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -15,6 +15,9 @@ class HttpWorkspaceManagerDAO(baseWorkspaceManagerUrl: String)(implicit val syst
 
   private def getApiClient(accessToken: String): ApiClient = {
     val client: ApiClient = new ApiClient()
+    // ignore any ServerConfigurations that pre-exist in the client by setting serverIndex to null
```

Comment:
Actually really helpful to know that's not default behavior. Thanks for digging into this!